generic/tests/cfg/lvm.cfg

```diff
@@ -11,7 +11,8 @@
     image_size_stg2 = 1G
     image_format_stg2 = qcow2
     guest_testdir = /mnt
-    kill_vm = no
+    mount_lv = yes
```

Comment:
This isn't meaningful, it should always be `yes`.

A better design is:

1. check whether the target lv device has been mounted
2. if not, perform mount